notebooks/MAST/TESS/beginner_tess_tap_search/beginner_tess_tap_search.ipynb

```diff
@@ -0,0 +1,375 @@
+    "### Step 1: Getting the footprint\n",
+    "\n",
+    "For our purposes, any one footprint related to a sector, camera, and chip combination is good enough. We are not currently accounting for small movements of the spacecraft to form a composite footprint. Observation IDs for this mission are constructed based on sector, camera, and chip combination, and we can use this to launch our footprint search:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "sector = '1'\n",
+    "camera = '1'\n",
+    "chip = '2'\n",
+    "observationIDwildcard = 'tess%-s000{}-{}-{}-%-s'.format(sector, camera, chip)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Here we query for a single ID and footprint to start with. For filters, we use the TESS mission, target types associated with light curves instead of full frame images, and the sector number. Note that the sector ID (sequence number) is in a different table in the CAOM database than most of the metadata we want, so we have to join these tables based on the shared observation ID.\n",
+    "\n",
+    "This query can use a synchronous job (with shorter timeouts) because we're only expecting a single line back from a query without time-consuming spatial search calculations or multiple joins of large tables. If you're having connection issues, try changing the launch function to *launch_job_async*."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "job = TAP_service.launch_job(\"\"\"\n",
+    "            SELECT top 1 obs_id, s_region  \n",
+    "            FROM dbo.caomobservation JOIN ivoa.obscore on dbo.caomobservation.observationID = ivoa.obscore.obs_id \n",
+    "            WHERE collection = 'TESS' and trgType = 'FIELD' and \n",
+    "            sequenceNumber = {} and\n",
+    "            observationID like '{}'\n",
+    "            \"\"\".format(sector, observationIDwildcard))\n",
+    "\n",
+    "#uncomment the line below to see the whole query with wildcards replaced\n",
+    "#print(job.parameters)\n",
+    "footprint_results = job.get_results()\n",
+    "footprint_results"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Step 2: Getting an inventory of TESS lightcurves within the footprint"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Here we take the footprint from the above query and find all lightcurves intersecting with this footprint, in ALL sectors. Depending on where this is in the sky, there could be responses only in the original sector, or there could be overlaps with other sectors. There would be more sector overlap near the poles, for instance. By filtering on obs_collection = TESS, we filter based on the TESS mission and exclude High Level Science Products even based on TESS.\n",
```

Comment:
"and exclude High Level Science Products even based on TESS," slightly confused about "even" in this sentence. Is it necessary? Maybe "and exclude High Level Science Products, even those based on TESS data" would be more clear?

Reply:
Thanks, wording fixed.